<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(170, 0, 0);</string>
  </property>
  <widget class="QLineEdit" name="textbox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 0);</string>
   </property>
   <property name="text">
    <string>Enter your guess number:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_new_game">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>New Game</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_res">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 255);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Check</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>89</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>0-100</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>270</y>
     <width>89</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>100-500</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_range">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>select range for game:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
